src/utils.js

```diff
@@ -119,55 +119,76 @@ module.exports = class Utils {
   static selectFiles(config = {}) {
     return new Promise((resolve, reject) => {
       /**
-       * Create a new INPUT element
-       * @type {HTMLElement}
+       * Check
        */
-      let inputElement = document.createElement('INPUT');
-
-      /**
-       * Set a 'FILE' type for this input element
-       * @type {string}
-       */
-      inputElement.type = 'file';
+      if (input === null) {
+        /**
+         * Create a new INPUT element
+         * @type {HTMLElement}
+         */
+        inputElement = document.createElement('INPUT');
 
-      if (config.multiple) {
-        inputElement.setAttribute('multiple', 'multiple');
-      }
+        /**
+         * Set a 'FILE' type for this input element
+         * @type {string}
+         */
+        inputElement.type = 'file';
 
-      if (config.accept) {
-        inputElement.setAttribute('accept', config.accept);
-      }
+        if (config.multiple) {
```

Comment:
config will be accepted only for the first calling?

Reply:
That's right.
I didn't consider that part.
I modified and committed.